insights/parsers/jboss_version.py

```diff
@@ -73,3 +88,35 @@ def release(self):
     def code_name(self):
         """string: code name of this running JBoss progress."""
         return self.parsed["code_name"]
+
+
+@parser(Specs.jboss_runtime_versions)
+class JbossRuntimeVersions(Parser):
+    """
+     Versions of multiple running JBoss - File ``data/insights_commands/jboss_versions``
+     ===================================================================================
+
+     This class is to access to file ``data/insights_commands/jboss_versions``
+
+     Typical content of file ``data/insights_commands/jboss_versions`` is::
+
+         {"/opt/jboss-datagrid-7.3.0-server": "Red Hat Data Grid - Version 7.3.0"}
+
+     This class parses the file content and stores data in the dict ``self.versions``.
+
+     Examples:
+         >>> len(all_jboss_versions.versions)
+         1
+         >>> all_jboss_versions.versions[0].major
+         7
+         >>> all_jboss_versions.versions[0].minor
+         3
+         >>> all_jboss_versions.versions[0].release
+         0
+     """
+
+    def parse_content(self, content):
+        self.versions = []
```

Comment:
In this case, it would be better to make the combiner as a list e.g. `JbossRuntimeVersions(Parser, list):`, instead of adding a new `self.versions` attribute.

Reply:
I update it as iterator.  As list has add, del etc operations, this class does not need these. So just use the iterator.